plugins/main/windows/win-bridge/win-bridge_windows.go

```diff
@@ -103,7 +103,7 @@ func cmdHnsAdd(args *skel.CmdArgs, n *NetConf) (*current.Result, error) {
 		return nil, fmt.Errorf("network %v not found", networkName)
 	}
 
-	if !strings.EqualFold(hnsNetwork.Type, "L2Bridge") {
+	if !strings.EqualFold(hnsNetwork.Type, "L2Bridge") || !strings.EqualFold(hnsNetwork.Type, "L2Tunnel") {
```

Comment:
Has a similar check in the function cmdHcnAdd. Should that be changed as well?

Reply:
Good Catch.